scripts/package-images

```diff
@@ -8,4 +8,5 @@ source ./scripts/version.sh
 mkdir -p dist/artifacts
 
 cp -f build/images.txt dist/artifacts/${PROG}-images.${PLATFORM}.txt
-gzip < build/images/${PROG}-airgap.tar > dist/artifacts/${PROG}-images.${PLATFORM}.tar.gz
+zstd -T0 -16 -f --long=25 build/images/${PROG}-airgap.tar -o dist/artifacts/${PROG}-images.${PLATFORM}.tar.zstd
+gzip -v -c build/images/${PROG}-airgap.tar > dist/artifacts/${PROG}-images.${PLATFORM}.tar.gz
```

Comment:
Would be a real nice to have for build times to run these in parallel but I realize that means job control, ugh.

Reply:
Yeah, same :/